
We do not want this package to grab the xemacs binary - we use xemacs-beta
for this preview package.

  - Rene Rebe <rene@exactcode.de>

--- xemacs/Makefile.in.in.orig	2005-01-06 13:29:23.346164720 +0100
+++ xemacs/Makefile.in.in	2005-01-06 16:44:26.293755448 +0100
@@ -398,11 +398,9 @@
 # ifdef CYGWIN
 	${INSTALL_PROGRAM} src/${PROGNAME} ${bindir}/${PROGNAME}-${version}.exe
 	-chmod 0755 ${bindir}/${PROGNAME}-${version}.exe
-	cd ${bindir} && $(RM) ./${PROGNAME} && ${LN_S} ${PROGNAME}-${version}.exe ./${PROGNAME}
 # else
 	${INSTALL_PROGRAM} src/${PROGNAME} ${bindir}/${PROGNAME}-${version}
 	-chmod 0755 ${bindir}/${PROGNAME}-${version}
-	cd ${bindir} && $(RM) ./${PROGNAME} && ${LN_S} ${PROGNAME}-${version} ./${PROGNAME}
 # endif /* CYGWIN */
 #endif /* WIN32_NATIVE */
 	if test "${prefix}" != "${exec_prefix}"; then \
